spec/features/events_spec.rb

```diff
@@ -152,7 +152,8 @@
 
   scenario "logged in as Organizer I can not change application status with radio buttons if the applications are locked" do
     login(:organizer)
-    @event.lock_application_status
+    @event.acceptances_have_been_sent = true
+    @event.rejections_have_been_sent = true
```

Comment:
Same as above. If the statuses are locked the radio buttons should not be shown (independent of the current phase, but currently the status can only be locked in execution phase). But maybe that will change later?